Conditional minion idea

dekanbro opened the pull request:
pass in a target address and data that has to complete for execute to run. if target address is 0, check that time is greater than conditionExecTime.

dekanbro committed:
Conditional minion idea

pass in a target address and data that has to complete for execute to run. if target address is 0, check that time is greater than conditionExecTime.

jordanlesich commented:
Would be sweet if the minion's was able to handle an 'else' or even 'else if' conditions as well, instead of only requires. 

ex. if(condition) call contract fn A
      else if (condition && conditionB) call fn B
      else call fn C

dekanbro committed:
Update MinionFactory.sol

remove else statement. can check if the time has passed add the conditional statement has executed

dekanbro commented:
> Would be sweet if the minion's was able to handle an 'else' or even 'else if' conditions as well, instead of only requires.
> 
> ex. if(condition) call contract fn A
> else if (condition && conditionB) call fn B
> else call fn C

Will have to think some more on this. Would not work as things are setup now because it just reverts the execute function, which could be rerun at a later time, like once a condition is passing
